tests/New_ATS/Tests/oneof-streaming/Python_client/ServerStreamingMethod_client.py

```diff
@@ -0,0 +1,51 @@
+import grpc
+import oneof_streaming_pb2  
+import oneof_streaming_pb2_grpc 
+import json
+import pytest
+import os
+
+def get_ServerStreamingMethod_output(test_input):
+    age = test_input['age']
+    message = test_input['request_oneof'].get('message')
+    
```

Comment:
add a comment that the test case json for this only has message as input for oneof message